templates/vue/src/components/lightbox/H5PIframe.vue

```diff
@@ -266,6 +264,7 @@ export default {
           h5pVideo.on("loaded", handleH5pAfterLoad)
         }
       }
+      this.$emit("is-loaded")
```

Comment:
Can you see if you can get rid of this event and use the `loaded` event instead that is already getting emitted at the right place? Unless they serve different functions - but it seems to me that this is emitting at the wrong time since technically it's not yet loaded until the `handleH5pAfterLoad` is called. (Please be careful with this and make sure the logic makes sense - I'm just observing the code without too much deep diving here)

Reply:
This event already existed in the H5P iframe and seems to play a role in loading the video - I moved it to the bottom of the function when dealing with another bug, but as it stands the event isn't used in this PR. I refactored the H5P progress loading to just use a vanilla i-frame.